src/fundus/publishers/ch/nzz.py

```diff
@@ -0,0 +1,49 @@
+import datetime
+import re
+from typing import List, Optional, Pattern
+
+from lxml.cssselect import CSSSelector
+
+from fundus.parser import ArticleBody, BaseParser, ParserProxy, attribute
+from fundus.parser.utility import (
+    apply_substitution_pattern_over_list,
+    extract_article_body_with_selector,
+    generic_author_parsing,
+    generic_date_parsing,
+)
+
+
+class NZZParser(ParserProxy):
+    class V1(BaseParser):
+        _summary_selector = CSSSelector("div.headline p.headline__lead")
+        _subheadline_selector = CSSSelector("div.article h2.subtitle, div.article h5.articlecomponent")
+        _paragraph_selector = CSSSelector(
```

Comment:
This [article](https://www.nzz.ch/international/usa-das-wichtigste-zum-rennen-um-das-weisse-haus-ld.1716364) has a table of contents, that is parsed as well.

Reply:
Also, as it seems this only occurs in English articles, but the last line with `Latest Articles` is parsed as well. See here: https://www.nzz.ch/english/us-election-2024-the-latest-polls-on-trump-and-biden-ld.1829123